front/app/utils/cl-router/Link.tsx

```diff
@@ -31,6 +32,7 @@ const Link = ({
   onlyActiveOnIndex,
   scrollToTop,
   onClick,
+  active: _active,
   ...otherProps
```

Comment:
Was getting really annoyed by this prop giving me console warnings. It is not an allowed prop on the `RouterLink` component, but somehow react doesn't check that properly when using spread syntax (`{ ...otherProps }`) which is why TS didn't detect it. Probably need to remove this prop everywhere it's used but maybe not worth spending time on. We could instead do this as part of the upcoming react router upgrade